source/lib/sharrplaybook-construct.ts

```diff
@@ -1,107 +1,102 @@
-#!/usr/bin/env node
-/*****************************************************************************
- *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.   *
- *                                                                            *
- *  Licensed under the Apache License, Version 2.0 (the "License"). You may   *
- *  not use this file except in compliance with the License. A copy of the    *
- *  License is located at                                                     *
- *                                                                            *
- *      http://www.apache.org/licenses/LICENSE-2.0                            *
- *                                                                            *
- *  or in the 'license' file accompanying this file. This file is distributed *
- *  on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,        *
- *  express or implied. See the License for the specific language governing   *
- *  permissions and limitations under the License.                            *
- *****************************************************************************/
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Did we miss to rename this file from the old solution name? `sharrplaybook-construct`

Reply:
There are still numerous references to the old solution name in this solution. Changing them all will have real consequences (not this one). For example, the SNS topic created for certain remediations may change, leading a user who has upgraded solutions to have two different destinations for notifications set up by the same remediation. Each change needs to be taken on a case-by-case basis. Not this one though, obviously the filename can safely be changed. But it will be a long time before the solution is consistent unless we decide to put a lot of effort into it.